docs/ref/utils.txt

```diff
@@ -974,14 +974,32 @@ appropriate entities.
 
     ``override`` is also usable as a function decorator.
 
-.. function:: localtime(value, timezone=None)
+.. function:: localtime(value=None, timezone=None)
 
     Converts an aware :class:`~datetime.datetime` to a different time zone,
     by default the :ref:`current time zone <default-current-time-zone>`.
 
+    When ``value`` is omitted, it defaults to :func:`now`.
+
     This function doesn't work on naive datetimes; use :func:`make_aware`
     instead.
 
+    .. versionchanged:: 1.11
+
+        The ``value`` argument now defaults to :func:`now`.
+
+.. function:: localdate(value=None, timezone=None)
+
+    .. versionadded:: 1.11
+
+    Converts an aware :class:`~datetime.datetime` to a different time zone, by
+    default the :ref:`current time zone <default-current-time-zone>`, using
```

Comment:
I tried to reword this sentence, see what you think.

Uses :func:`localtime` to convert an aware :class:`~datetime.datetime` to a :meth:`~datetime.datetime.date` in a different time zone, by default the :ref:`current time zone <default-current-time-zone>`.